0 Products/80 OpenGrab EPM v3/0 Main Page.md

```diff
@@ -59,6 +65,11 @@ when the magnet is turning on, causing an insulation breakdown.
 
 The diagrams below document the mechanical arrangement and dimensions (click to enlarge):
 
+### R5C and newer
+<img src="Pinout_R5C.png" width=350 title="Top view">
+<img src="Dim_R5C.png" width=350 title="Drawing">
+
+### R4B and older
```

Comment:
I feel like the word "Version" is missing in the headings. Like, "Version R5C and newer" makes more sense. Also, the images need to be sized correctly. And I just noticed that the current version has this issue too: when arranging images horizontally, define their *height*, not *width*. Like that:

```
<img src="Pinout_R5C.png" height=400 title="Top view">
<img src="Dim_R5C.png" height=400 title="Drawing">
```

I'm not sure if 400 px is okay, it's just to give you the idea.